fastlane/lib/fastlane/fast_file.rb

```diff
@@ -175,6 +175,7 @@ def actions_path(path)
 
     # Execute shell command
     def sh(command, log: true, error_callback: nil)
+      command = command.map(&:to_s).map(&:shellescape).join(" ") if command.kind_of?(Array)
```

Comment:
This repeats what is in the `Actions.sh` method, but it makes the `command_header` come out right.

Reply:
Can you refactor this so that the conversion is a separate method? As far as I can see the same logic is repeated here, once again at `fastlane/lib/fastlane/helper/sh_helper.rb:30` and one more time at `fastlane_core/lib/fastlane_core/configuration/config_item.rb:161`